docs/specs/metadata.yml

```diff
@@ -28,8 +28,113 @@ inputs:
     # Title
 outputs:
   docs/a.json: |
-    { "ms.custom-metadata": null, "author":"xxx", "uid": null, "breadcrumb_path": "/breadcrumb_path", "title": "Title"}
+    { 
+      "ms.custom-metadata": null, 
+      "author":"xxx", 
+      "uid": null, 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",    
+    }
+---
+# output _xrefmap per file to structure json
+repos:
+  https://docs.com/test#test:
+    - files:
+        docfx.yml:
+        docs/a.md: |
+          ---
+          uid: abc
+          breadcrumb_path: "/breadcrumb_path"
+          author: xxx
+          ---
+          # Title
+outputs:
+  docs/a.json: |
+    {      
+      "author":"xxx", 
+      "uid": "abc", 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",  
+      "_xrefmap": {
+          "abc":{
+            "href": "https://docs.com/docs/a?branch=test",
+            "name": "abc",
+            "uid": "abc",
+            "specType": "uid",
+          }
+      }
+    }
+  .xrefmap.json: | 
+    {
+      "references":[
+        {"uid": "abc", "name": "abc" ,"href": "https://docs.com/docs/a?branch=test","specType": undefined },       
+      ]
+    }
+---
+# In continue build, output xrefmap to page metadata but not page data
+continueBuildOnly: true
+repos:
+  https://docs.com/test#test:
+    - files:
+        docfx.yml: |
+          outputType: pageJson
+        docs/a.md: |
+          ---
+          uid: abc
+          breadcrumb_path: "/breadcrumb_path"
+          author: xxx
+          ---
+          # Title
+outputs:  
+  docs/a.raw.page.json: | 
+    { 
+      "rawMetadata": {
+        "xrefmap": undefined
+      }
+    }
+  docs/a.mta.json: |
+    {       
+      "xrefmap": "[{\"uid\":\"abc\",\"href\":\"https://docs.com/docs/a?branch=test\",\"specType\":\"uid\",\"name\":\"abc\"}]"
```

Comment:
Renaming and setting this value to `.mta.json` is typically done in the extensibility model. see [serviceData example](https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=/packages/docfx-templates/ModuleUnit.mta.json.js&version=GBdevelop&line=154&lineEnd=155&lineStartColumn=1&lineEndColumn=40&lineStyle=plain&_a=contents) Docfx itself should generally define what the `.json` file looks like, the `.mta.json.js` and `.raw.page.js` scripts determines what ends up in `.mta.json` and `.raw.page.json`.

Reply:
@yufeih  as synced in meeting today, I guess we don't do this change for now, right? 
Please help resolve conversation if you feel it is OK for you.